.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '42 5 * * *'
```

Comment:
Remove these lines.

I read your comments on https://github.com/szabgab/test-class/blob/master/.github/workflows/ci.yml and watch your video https://perlmaven.com/github-actions-ci-job-for-test-class.

cpantesters covers the issue for CPAN modules.